pkg/virtctl/vnc/vnc.go

```diff
@@ -84,37 +84,51 @@ type VNC struct {
 	clientConfig clientcmd.ClientConfig
 }
 
-func (o *VNC) Run(cmd *cobra.Command, args []string) error {
-	namespace, _, err := o.clientConfig.Namespace()
+func (o *VNC) setupLocalProxy(vmiName string) (*net.TCPListener, error) {
```

Comment:
Nit: I think a createXXX  would be better name here as you are actually creating a new listener here.

Reply:
@ezrasilvera Done, going with the create* approach